src/Nethermind/Nethermind.Trie/TrieNode.cs

```diff
@@ -526,9 +527,34 @@ public void ResolveKey(ITrieNodeResolver tree, bool isRoot, ICappedArrayPool? bu
             return null;
         }
 
-        internal CappedArray<byte> RlpEncode(ITrieNodeResolver tree, ICappedArrayPool? bufferPool = null)
+        /*
```

Comment:
Remove commented out block?

Reply:
Hmm.. I can't remember why I commented it out. Oh well...